### PR TITLE
Update Danish Translation (93% / 181)

### PR DESCRIPTION
**Pull request by Stisen1**
- Focused on editor-related translations
- Small changes to campaignbonus descriptions (too long strings showed "..." symbols)
- Small changes to some grammatic articles
- Renamed some artifacts to names that seemes a bit more relatable to danish gamers
- Changes to hero names to make them more relatable to danish gamers.

### CODE REVIEW

File: files/lang/dk.po [hunk 1]
```diff
@@ -1628,34 +1624,34 @@ msgid "campaignBonus|Dragon Sword"
 msgstr "Dragesværdet"
 
 msgid "campaignBonus|Fizbin Medal"
-msgstr "Fizbin Ulykkesmedaljen"
+msgstr "Ulykkesmedalje"
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:1627]: You don't learn what Fizbin means in this game. It's just a name without its original context. And why should "medal of misfortune" not draw your attention as a newbie?

[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:1627]: Okay. What about calling it Fizbin-medalje then? I mean in the string we are talking about here.

[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:1627]: Nah. I see your point now. It needs to be secretive and intreaging enough to let players take the bate, just to realize during the first few turns, that they made a horrible mistake 😀

File: files/lang/dk.po [hunk 2]
```diff
@@ -384,15 +384,17 @@ msgstr "Hastighed: %{speed}"
 msgid "Speed"
 msgstr "Hastighed"
 
+msgid "Army Order"
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po]: Oh... okay. I have downloaded the recently changed pre-release dk.mo-file, and then I tried to manually change my own copy of the dk.po-file to look like GitHub Desktop tell me it should look like. But I guess I misunderstood something. Also... I checked how "turn order" looks like inside the game, and then I discovered some line-issues with other options. These have been changed too.

File: files/lang/dk.po [hunk 3]
```diff
@@ -424,9 +425,8 @@ msgstr "Skadeinfo"
 msgid "Set the speed of combat actions and animations."
 msgstr "Indstil hastighed for kamphandlinger og animationer."
 
-#, fuzzy
 msgid "Toggle to display the turn order during the battle."
-msgstr "Vælg om kampens hærorden skal vises eller ej."
+msgstr "Vælg om kampens tur-orden skal vises eller ej."
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po]: It's spelled as "turorden" both places now

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@Stisen1 , thank you so much for the constant updates of the translation!

**Pull request merged**